app/src/main/java/org/gdg/frisbee/android/arrow/SummitNotificationReceiver.java

```diff
@@ -4,6 +4,9 @@
 import android.content.Intent;
 import android.support.v4.content.WakefulBroadcastReceiver;
 
+import org.gdg.frisbee.android.Const;
+import org.gdg.frisbee.android.eventseries.TaggedEventSeriesNotificationService;
+
 public class SummitNotificationReceiver extends WakefulBroadcastReceiver {
```

Comment:
Should we rename this as well then? 
And also, we can then move these classes to `eventseries` package.